### ISSUE TITLE
Sort tests on speed

### ISSUE THREAD

**Issue by hugo-vrijswijk** [stryker-mutator/stryker4s#16]
Run the fastest tests first for each mutation. This way (hopefully) the tests cause a killed mutant sooner

**Comment by nicojs** [comment]
This seems like a complicated thing to do. Maybe impact analysis and multi-threaded runs should go first?

**Comment by hugo-vrijswijk** [comment]
Definitely. I've created a [Waffle.io board](https://waffle.io/stryker-mutator/stryker4s) and sorted the issues on what I think should go first

**Comment by nicojs** [comment]
It would if you have some integration tests intermingled with your unit tests 🤷‍♀️
